src/vs/editor/common/modes/languageConfigurationRegistry.ts

```diff
@@ -46,18 +46,20 @@ export interface IIndentConverter {
 export class RichEditSupport {
 
 	private readonly _conf: LanguageConfiguration;
+	private readonly _languageIdentifier: LanguageIdentifier;
+	private _brackets: RichEditBrackets;
+	private _electricCharacter: BracketElectricCharacterSupport;
```

Comment:
Same as above.

Can you please add `this._brackets = null; this._electricCharacter = null;`